eng/templates/variables-template.yml

```diff
@@ -0,0 +1,26 @@
+# Variables used for both the CI and the official builds.
+
+- name: _TeamName
+  value: DotNetInteractive
+- name: _BuildConfig
+  value: Release
+- name: _PublishUsingPipelines
+  value: true
+- name: _DotNetArtifactsCategory
+  value: .NETCore
+- name: DisableDockerDetector
+  value: true
+- name: PocketLoggerLogPath
+  value: $(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)
```

Comment:
I'm trying to move this under the TestResults folder so that they get included in the test artifacts output.

Reply:
perfect, will make that change. I'll likely look through the changes you have going in your PR and pull them in so they are all intact.